share/completions/apm.fish

```diff
@@ -61,7 +61,7 @@ end
 
 # Lists all apm config items
 function __fish_apm_config_items
-    apm config list | grep -v '^;\|^$' | tr "\ =\ " "\t"
+    apm config list | string replace -r "\s*=\s*" "=" | string match -r -v '^;|^$' | tr "=" "\t"
```

Comment:
I used this example output. Let me get back to you I'm on my way to work.

```
; cli configs
globalconfig = "C:\\Users\\chaines\\AppData\\Local\\atom\\app-0.177.0\\resources\\app\\apm\\.apmrc"
user-agent = "npm/2.3.0 node/v0.10.35 win32 ia32"
userconfig = "C:\\Users\\chaines\\.atom\\.apmrc"

; userconfig C:\Users\chaines\.atom\.apmrc
https-proxy = "<MASKED>"
proxy = "<MASKED>"
strict-ssl = false
```

Let me get back to you.